test/unit/org/apache/cassandra/tools/nodetool/SetAuthCacheConfigTest.java

```diff
@@ -182,6 +197,26 @@ private void assertSetConfig(AuthCache<?, ?> authCache, String cacheName)
         assertThat(authCache.getActiveUpdate()).isFalse();
     }
 
+    private void assertSetConfigDisabled(AuthCache<?, ?> authCache, String cacheName)
+    {
+        ToolRunner.ToolResult tool = ToolRunner.invokeNodetool("setauthcacheconfig",
+                                                               "--cache-name", cacheName,
+                                                               "--validity-period", "1",
+                                                               "--update-interval", "-1",
+                                                               "--max-entries", "3",
+                                                               "--disable-active-update");
+        tool.assertOnCleanExit();
+        assertThat(tool.getStdout()).isEqualTo("Changed Validity Period to 1\n" +
+                                               "Changed Update Interval to -1\n" +
+                                               "Changed Max Entries to 3\n" +
+                                               "Changed Active Update to false\n");
+
+        assertThat(authCache.getValidity()).isEqualTo(1);
+        assertThat(authCache.getUpdateInterval()).isEqualTo(1);
```

Comment:
Sure I also updated `setPermissionsUpdateInterval` adding explicit checks, exceptions
https://github.com/ekaterinadimitrova2/cassandra/pull/194/files#diff-054af65b8d690b0fddc3e0a4ef05a80d8f1d6689b4f77912795fec019200666cR1360-R1365
Just stressing on that in case you see some issue with doing it that way /CC @dcapwell